docs/topics/native/native-objc-interop.md

```diff
@@ -208,11 +208,17 @@ Learn more about the [`async`/`await` mechanism in Swift](https://docs.swift.org
 
 ### Extensions and category members
 
-Members of Objective-C categories and Swift extensions are imported to Kotlin
+Members of Objective-C categories and Swift extensions are generally imported to Kotlin
 as extensions. That's why these declarations can't be overridden in Kotlin.
 And the extension initializers aren't available as Kotlin constructors.
 
-Kotlin extensions to "regular" Kotlin classes are imported to Swift and Objective-C as extensions and category members respectively.
+> Currently, there are two exceptions. Starting with Kotlin 1.8.20, category members that are declared
+> in the same headers as the NSView class (from the AppKit framework) and UIView classes (from the UIKit framework) are
+> imported as members of these classes. It means that you can override methods subclassing from NSView and UIView.
```

Comment:
```suggestion
> imported as members of these classes. This means that you can override methods when subclassing from NSView and UIView.
```

Reply:
The initial idea was to say "you can override methods that subclass from NSView and UIView." but maybe your variant works too